poweredup.ts

```diff
@@ -16,7 +16,6 @@ import { EventEmitter } from "events";
 import Debug = require("debug");
 const debug = Debug("PoweredUP");
 import noble = require("noble-mac");
-import { start } from "repl";
```

Comment:
Thank you, this PR looks good! I hadn't even noticed that this had gotten in here. I assume my IDE inserted it at some point without me realising...

Reply:
Who knows? I just removed it because my IDE showed it wasn't actually being used anywhere.